### DISCUSSION TITLE
How to position the message and pupup windows to the lower right corner of the UI?

### DISCUSSION THREAD

**Discussion by hopezh** [folke/noice.nvim#469] [Q&A]
By default, the message and popup windows show in the upper-right corner, as shown below. 

May I ask how to move them to the lower-right corner, or any other position, of the UI? 

![image](https://github.com/folke/noice.nvim/assets/23435810/9e2238e3-43a4-48a9-9935-ae046ba7acc9)

**Comment by Jayphen** [comment]
I would love to know as well. I tried the following, but it did not work:

```
      views = {
        notify = {
          merge = true,
          timeout = 3000,
          render = "minimal",
          stages = "fade",
          top_down = false,
        },
      },
```

**Comment by ilias777** [comment]
Set `top_down = false` in the notify options and not in the noice options.

Try this:

```lua
return {
    'rcarriga/nvim-notify',
    opts = {
        -- other settings
        top_down = false,
        -- other settings
    },
}
```

**Reply by mikavilpas** [reply]
This worked! Thanks so much!
